### PR TITLE
modified home page

### PR DESCRIPTION
**Pull request by Edris01**
i was modifying the home page and filling in the content

### CODE REVIEW

File: assets/css/style.css [hunk 1]
```diff
@@ -87,11 +87,10 @@ input{
...
 }
 section{
-    padding: 20px 20px 0px 20px;
-    background-image: url(../image/corrupt.jpg);
-    /*background-size: 100% 100%;*/
-    background-repeat: repeat-x;
-    background-attachment: fixed;
+    padding: 10px 10px 0px 20px;
+    display: grid;
+    grid-template-rows: auto 40%;
```
Comment (neelxie) [inline assets/css/style.css:92]: The side of the image cnt not be auto, make it at list 40% auto

File: assets/css/style.css [hunk 2]
```diff
@@ -120,8 +119,50 @@ q{
...
 legend{
-    color: #f5f0f0;
+    color: #f01111;
     font-weight: bold;
     font-style: oblique;
     font-size: 30px;
+}
+.home0{
+    display: grid;
+    grid-template-columns: auto 60%;
```
Comment (neelxie) [inline assets/css/style.css:129]: side of the image can not be auto, at least say 40% auto

File: assets/css/style.css [hunk 3]
```diff
@@ -120,8 +119,50 @@ q{
...
+    align-items: center;
+    margin: 0;
```
Comment (neelxie) [inline assets/css/style.css:131]: this has no effect

File: assets/css/style.css [hunk 4]
```diff
@@ -120,8 +119,50 @@ q{
...
+}
+.home1{
+    align-items: center;
+    min-height: auto;
+}
+.home2{
+    width: 500px;
```
Comment (neelxie) [inline assets/css/style.css:138]: make height and width 100%

File: assets/css/style.css [hunk 5]
```diff
@@ -120,8 +119,50 @@ q{
...
+    height: 510px;
+    border-radius: 30px;
+}
+.home3{
+    text-align: left;
+    font-size: 20px;
+    font-family: sans-serif;
+    padding: 0px;
```
Comment (neelxie) [inline assets/css/style.css:146]: remove padding and margin here, it has no effect

File: assets/css/style.css (hunk 2 above)
Comment (neelxie) [reply assets/css/style.css:129]: add a column gap here

**Comment by neelxie** [comment]
About images, set the dimensions for the parent and elements so that u do not explicitly set their height and width so they can easily scale on any screen size.

File: assets/css/style.css (hunk 4 above)
Comment (Edris01) [reply assets/css/style.css:138]: when i assigned the image with the width and height of 100%

**Pull request merged**